src/child/containers/sidebars/favourites/Favourites.js

```diff
@@ -48,6 +48,15 @@ class Favourites extends Component {
         });
     }
 
+    componentWillReceiveProps(nextProps) {
+        // onDragEnd is not called when a tile is successfully dragged into
```

Comment:
I think it _should_ be called - is it definitely not being called? And if so, any idea why?